examples/navigation/epics/clearSearchResults.js

```diff
@@ -0,0 +1,11 @@
+import { LOCATION_CHANGE } from 'react-router-redux';
+import * as ActionTypes from '../actionTypes';
+
+export default action$ =>
+  action$.ofType(LOCATION_CHANGE)
+    .filter(({ payload }) =>
+      payload.pathname === '/' && !!!payload.query.q
+    )
+    .mapTo({
+      type: ActionTypes.CLEARED_RESULTS
+    });
```

Comment:
Something to note here about `mapTo`, it will literally emit the same instance of the object over and over. If anything downstream is checking to see if the action is a new instance, it could have consequences. In contrast, `map(() => { foo: 'bar' })` will emit a new object instance each time.